playground-authjs/server/api/auth/[...].ts

```diff
@@ -1,18 +1,18 @@
-import CredentialsProvider from 'next-auth/providers/credentials'
-import GithubProvider from 'next-auth/providers/github'
+import CredentialsProvider from '@auth/core/providers/credentials'
+import GithubProvider from '@auth/core/providers/github'
 import { NuxtAuthHandler } from '#auth'
 
 export default NuxtAuthHandler({
   // a) Never hardcode your secret in your code!! and b) use a secure secret, `test-123` is **not** secure!!
   secret: process.env.AUTH_SECRET ?? 'test-123',
   providers: [
-    // @ts-expect-error You need to use .default here for it to work during SSR. May be fixed via Vite at some point
-    GithubProvider.default({
+    // // @ts-expect-error You need to use .default here for it to work during SSR. May be fixed via Vite at some point
```

Comment:
I think these comments can now fully go (finally!)?

Reply:
@zoey-kaiser We would need to write in a migration guide to remove `.default` from the NuxtAuthHandler